src/components/dropdown/index.coffee

```diff
@@ -53,8 +58,15 @@ positionDropdown = ({alignments, selection, dropdown, useScroll}, {matchWidth})
     dropdown.style('overflow-y','auto')
 
   dropdown
+    .classed('hx-dropdown-up', direction is 'up')
+    .classed('hx-dropdown-down', direction is 'down')
+    .classed('hx-dropdown-left', direction is 'left')
+    .classed('hx-dropdown-right', direction is 'right')
+    .attr('data-direction', direction)
+    .style('top', 'auto')
+    .style('bottom', undefined)
```

Comment:
We need `top` of `auto` to override the CSS style of `-10000px` and `bottom` of `undefined` to clear the bottom value when switching between `up` and `down`